types/fulu/blob_sidecar.yaml

```
@@ -0,0 +1,8 @@
+Fulu:
+  BlobSidecars:
+    type: array
+    items:
+      $ref: "#/Deneb/BlobSidecar"
+    minItems: 0
+    maxItems: 48 # placeholder, depending on the final value we chose and BPO schedule
```

Hmm I feel that we should just use a really high value here instead. Let's discuss this first.
```suggestion
    maxItems: 4096  # MAX_BLOB_COMMITMENTS_PER_BLOCK
```

---

+1 on setting it high enough that we don't ever need to change it anymore

---

there was some discussion in https://github.com/ethereum/beacon-APIs/pull/488 on this previously, I am generally in favor of using sane list limits but since blob limit increase will be effectively decoupled from (named) forks with EIP-7892 I also think it's best to use 4096 now

---

+1 for 4096

---

seems like we have enough consensus here, updated